java-parse/README.md

```diff
@@ -54,6 +54,81 @@ And here's a sample output:
 
 ![graph of first climbstairs function][climbstairs_graph]
 
+## Printing a JSON Implication Graph
+
+```
+Print out implication graph (in JSON) of many classes and methods.
+
+  main.byte print-implication
+
+=== flags ===
+
+  [-c class]   file to analyze (put before method name).
+  [-m method]  name in classfile to target.
+  [-o output]  to dot instead of json and specify out folder.
```

Comment:
If you include this option you get a `.dot` file which you can turn into an image and include in the docs. Could be cool.